ra/ra.go

```diff
@@ -2444,10 +2445,12 @@ func (ra *RegistrationAuthorityImpl) NewOrder(ctx context.Context, req *rapb.New
 		return existingOrder, nil
 	}
 
-	// Check if there is rate limit space for issuing a certificate.
-	err = ra.checkNewOrderLimits(ctx, newOrder.Names, newOrder.RegistrationID)
-	if err != nil {
-		return nil, err
+	if !req.LimitsExempt {
```

Comment:
Add a comment that this exists because we prioritize and encourage replacing certificates when they fall within the ARI window.

Reply:
Thanks. I've gone ahead and done this for the lower line as well.